tests/test_uniquegift.py

```diff
@@ -410,6 +416,8 @@ class UniqueGiftInfoTestBase:
     origin = UniqueGiftInfo.UPGRADE
     owned_gift_id = "some_id"
     transfer_star_count = 10
+    last_resale_star_count = 5
+    next_transfer_date = dtm.datetime.now(tz=UTC).replace(microsecond=0)
```

Comment:
please also add a new test `de_json_timezone_locatization` that makes sure that `next_transfer_data` is localized correctly :) there are already corresponding test cases in other classes that you can copy from